tabular/src/autogluon/tabular/features/feature_metadata.py

```diff
@@ -369,7 +371,19 @@ def print_feature_metadata_full(self, log_prefix='', print_only_one_special=Fals
             return output_str
 
     @classmethod
```

Comment:
Docs for the FeatureMetadata class could still be improved, eg. this description of the object: 
"Feature metadata contains information about features that are not directly apparent in the raw data itself."
could be clarified as:
"A FeatureMetadata object contains type information about features, which may not be apparent from the raw data itself (eg. whether a column contains numerical, categorical, or text values)."

Reply:
Will address in follow-up

Reply:
Will address in follow-up